requirements.txt

```diff
@@ -1,5 +1,5 @@
-scipy==1.7.3
-matplotlib==3.5.3
-pyaudio>=0.2.11  # Portaudio is required beforehand, for linux: sudo apt-get install libasound-dev
+scipy>=1.7.3
+matplotlib>=3.5.3
+pyaudio>=0.2.12  # Portaudio is required beforehand, for linux: sudo apt-get install libasound-dev
```

Comment:
why was '>=0.2.11' not enough again?

Reply:
I wouldn't be able to play audio with 0.2.11 on my M1 with Python3.10. Getting `SystemError: PY_SSIZE_T_CLEAN macro must be defined for '#' formats`

Reply:
To avoid breaking the conda worfklow for Python 3.9 and prior, I'd suggest:

```
pyaudio>=0.2.12; python_version >= '3.10'
pyaudio>=0.2.11; python_version < '3.10'
```

instead. For Python 3.10+ it cannot be helped (for now).

Reply:
Using 0.2.11, pytest will still be green because all the xfail in aserver and arecorder when there are real devices are actually failing.

Reply:
the more tests that can be run reliable during GHA the better. Accepted failing should be an exception.

Reply:
Those tests have the precondition of having real device. So a better should be skipif instead of xfail. We can change that in the next release, which I plan to do some refactor works